### ISSUE TITLE
it always says this test is secured

### ISSUE THREAD

**Issue by bruh2345df** [numero69/edupage_hack#4]

**Comment by NotRixard** [comment]
Use this: https://github.com/RichardKanshen/edupagehackextended

**Comment by bruh2345df** [comment]
didint change anything

> Use this: https://github.com/RichardKanshen/edupagehackextended

**Comment by NotRixard** [comment]
Did you follow the tutorial?

**Comment by bruh2345df** [comment]
Is it not the same as this one here?

**Comment by NotRixard** [comment]
It's a very customized fork, if you want to use it on secured tests you have to make a proxy, it's all explained in the tutorial.

**Comment by bruh2345df** [comment]
All i see is follow the instructions for the proxy but the tut is all about requestly

**Comment by NotRixard** [comment]
Listen, just do what the tutorial says, that's all.

**Comment by bruh2345df** [comment]
yeah i found it still having trouble some files are missing like the empty file and other stuff

**Comment by NotRixard** [comment]
If you have trouble then make an issue report on the fork. Hopefully the guy may help with whatever problem you're having.

**Comment by RichardKanshen** [comment]
> yeah i found it still having trouble some files are missing like the empty file and other stuff

You are getting an empty file?
